frontend/public/components/machine.tsx

```diff
@@ -37,7 +37,8 @@ const tableColumnClasses = [
   classNames('col-lg-2', 'col-md-3', 'col-sm-3', 'hidden-xs'),
   classNames('col-lg-2', 'col-md-3', 'col-sm-3', 'hidden-xs'),
   classNames('col-lg-2', 'hidden-md', 'hidden-sm', 'hidden-xs'),
-  classNames('col-lg-2', 'hidden-md', 'hidden-sm', 'hidden-xs'),
+  classNames('col-lg-1', 'hidden-md', 'hidden-sm', 'hidden-xs'),
+  classNames('col-lg-1', 'hidden-md', 'hidden-sm', 'hidden-xs'),
```

Comment:
We probably want to remove the grid column classes instead of `col-lg-1` as in #3669.